### ISSUE TITLE
Sign Git commits

### ISSUE THREAD

**Issue by ElvishJerricco** [mssun/passforios#246]
Git supports GPG signing of commits. I use this on my password repo due to its sensitive nature. It'd be great if I could do this for passwords generated on Pass for iOS.

**Comment by mssun** [comment]
I agree with you, but we rely on the [objective-git](https://github.com/libgit2/objective-git) project currently which does not support GPG commit signing (https://github.com/libgit2/objective-git/issues/687) right now. We have to either wait for this feature or directly use API (`git_commit_create_with_signature`) provided by `libgit2`.

**Comment by ElvishJerricco** [comment]
Hm is it easy to use libgit2 in an objective-git codebase? Or would it require doing a bunch of things again the libgit2 way?
